### ISSUE TITLE
Keep Variations In Skirmish

### ISSUE THREAD

**Issue by ilosevsrat** [Sh0wdown/UnofficialCrusaderPatch#633]
so, is there any problems  making different lords use different keep size in skirmish?   Like, Big keep would definitely fit Wolf and Frederick.   :)

maybe its difficult to implement, just asking :3rd_place_medal:

**Comment by Halbarad1** [comment]
This would of course just be for peoples own aiv since the different keep size wouldn't fit with the original aiv. However if it could work that could be really nice and make new AIs more personal.

**Comment by ilosevsrat** [comment]
> This would of course just be for peoples own aiv since the different keep size wouldn't fit with the original aiv. However if it could work that could be really nice and make new AIs more personal.

yes, you are right,  Large Keep will definitely screw Granary placement in 90% of the time for AI =D but Small AiV tweaks will solve this problem. also i imagine How cool it will be to see WOlf with LArge keep :3
